src/mlpack/methods/range_search/rs_model.hpp

```diff
@@ -385,6 +386,10 @@ class RSModel
    */
   void CleanMemory();
 };
+std::string LoadModel(std::string s);
+void SaveModel(RSModel* model,std::string s);
+//Serialize 2 models and then check their equality
+bool CheckModelSerial(RSModel* , RSModel* );
```

Comment:
These are only used in the tests, so I would suggest that any test-specific functionality be put there and not here.